README_py.md

```diff
@@ -0,0 +1,26 @@
+# Welcome to a Toy PyBind11 Algorithm
+
+## Why?
+The goal of this project is to allow users to take an algorithm written in C++ and translate that into runnable Python code.
+This could allow a user to develop C++ code and run it on the EmotiBit's firmware, software, and through Python hitting three different metrics.
+
+## Installation Steps
+* Follow ['this link'](https://github.com/EmotiBit/EmotiBit_Biometric_Lib/tree/master/py) for details on how to setup a basic Anaconda environment.
+  * Everything up to downloading Spyder is a necessary step in the process. Skip downloading the Anaconda environment given, you can find the correct one in the GitHub repo below.
```

Comment:
I think a "con" of sending people to the link above is that they might just follow the complete guide and also install the other environment. It's hard to define a "when you reach this part of the doc, stop and come back".
Maybe the better alternative is
1. install anaconda using [anaconda's official guide]
2. import environment (link to anaconda's guide) and use the `yml` file we provide